setup.py

```diff
@@ -15,6 +15,7 @@ def install_requires():
     return [
         'pandas',
         'requests',
+        'tenacity',
```

Comment:
@jbredeche do you have a goal for how minimal dependencies should be for this library?

Reply:
my 2c is that I think we should keep dependencies of this lib to a minimum to the extent possible. Especially given that requests already has a relatively well-supported interface for retries, my preference would be to drop this if possible.